tests/unit/storage/test_gcp.py

```diff
@@ -100,13 +152,15 @@ def test_manually_then_read_then_write_then_read_file(self, codecov_vcr):
         blob = storage.get_blob(bucket_name, path)
         blob.reload()
         assert blob.content_type == "text/plain"
-        assert blob.content_encoding == "gzip"
+        assert blob.content_encoding == "zstd"
 
     def test_write_then_read_file_gzipped(self, codecov_vcr):
         storage = GCPStorageService(gcp_config)
         path = "test_write_then_read_file/result"
-        data = gzip.compress("lorem ipsum dolor test_write_then_read_file á".encode())
-        bucket_name = "testingarchive02"
+        data = zstandard.compress(
+            "lorem ipsum dolor test_write_then_read_file á".encode()
+        )
+        bucket_name = "testingarchive1"
         writing_result = storage.write_file(
             bucket_name, path, data, is_already_gzipped=True
```

Comment:
I would rather keep the meaning of the `is_already_gzipped` flag, and assume it is indeed `gzip`, as some API users might use it that way.

Reply:
having a separate `is_already_zstded` feels bad to me because then would we need some validation to make sure that both `is_already_gzipped` and `is_already_zstded` are both not set to True, and to me it would just make the interface more complicated

i did want to make this: `is_already_compressed` but that would require a bunch of changes wherever this is being used because it's a kwarg